src/settings-tab.ts

```diff
@@ -178,6 +193,7 @@ export class MindMapSettingsTab extends PluginSettingTab {
           })
       );
 
+    const color_3 =
     new Setting(containerEl)
```

Comment:
- move this new Setting to the previous line, right after the `=`, followed by a space char.